### PR TITLE
feat(generate-loader): scaffold in a specified path

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
Enhancement

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Nope

**Summary**
Added the ability to scaffold `loader` template in a specified path. Addresses #2021

**Does this PR introduce a breaking change?**
Nope

**Other information**
https://github.com/webpack/webpack-cli/issues/2021#issuecomment-722395490

### CODE REVIEW

**Review by snitin315 — COMMENTED** [review]
how do we have to provide that specified path ?

**Comment by jamesgeorge007** [comment]
> how do we have to provide that specified path?

`webpack-cli loader [path]`

**Comment by snitin315** [comment]
Lets document it.

**Review by evilebottnawi — APPROVED** [review]

**Review by snitin315 — DISMISSED** [review]

File: packages/generate-loader/README.md [hunk 1]
```diff
@@ -27,7 +27,13 @@ generateLoader();
 ### CLI (via `webpack-cli`)
 
 ```bash
-npx webpack-cli generate-loader
+npx webpack-cli loader
+```
+
+> Optionally specify a path for generating the loader template.
```
Comment (anshumanv) [inline packages/generate-loader/README.md:33]: should mention both absolute and relative path work

Comment (anshumanv) [reply packages/generate-loader/README.md:33]: maybe add test

**Review by anshumanv — COMMENTED** [review]
Can we add for https://github.com/webpack/webpack-cli/pull/2073 too? Since I missed reviewing it before merge

**Review by evilebottnawi — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Comment by jamesgeorge007** [comment]
> /cc @anshumanv

Oops, merged

**Pull request merged**